README.md

```diff
@@ -653,7 +653,7 @@ err := descopeClient.Management.Tenant().Update(context.Background(), "my-custom
 
 // Tenant deletion cannot be undone. Use carefully.
 // Pass true to cascade value, in case you want to delete all users/keys associated only with this tenant
-err := descopeClient.Management.Tenant().Delete(context.Background(), "my-custom-id", false)
+err := descopeClient.Management.Tenant().Delete(context.Background(), "my-custom-id", true)
```

Comment:
Why do you want to change this ? 
false is also a valid value, and we document that you can pass true  as well

Reply:
other SDKs have different values here, i am going to change them to false (default behaviour)